docs/spelling_wordlist.txt

```diff
@@ -33,6 +33,7 @@ deserialized
 deserializers
 deserializes
 deserializing
+e
```

Comment:
You shouldn't need to do this. These are from "i.e.", right? Travis correctly considers it a word.

Are you developing locally on OS X? If so, the master branch of pyenchant has more up-to-date dictionaries. It hasn't been released though :(